doc/rose-rug-task-run.html

```diff
@@ -692,6 +692,114 @@ <h2 id="rose-task-run.util.rose_arch">Built-in Application: rose_arch</h2>
 # ...
 </pre>
 
+    <h2 id="rose-task-run.util.rose_bunch">Built-in Application:rose_bunch</h2>
+
+    <p>This built-in application allows running of multiple command variants in
+    parallel under a single job, as defined by the application
+    configuration.</p>
+
+    <p>The application is normally configured in the
+    <kbd>[bunch-settings]</kbd> and <kbd>[bunch-args]</kbd> sections in
+    <var>rose-app.conf</var>.</p>
+
+    <p>Each variant of the command is run in the same working directory with
+    its output directed to separate <var>.out</var> and <var>.err</var> files
+    of the form <var>bunch.&lt;name&gt;.out</var>. Should you need separate working
+    directories your should configure your command to create the appropriate
+    subdirectory for working in.</p>
+
+    <p>Note that, when running under batch submission systems, you will need
+    to set resource requests to reflect the resources required by running
+    multiple commands at once e.g. if one command would require 1GB memory and
+    you have configured your app to run up to 4 commands at once then you will
+    need to request 4GB of memory.</p>
+
+    <p>The following <kbd>[bunch-settings]</kbd> settings are accepted:</p>
```

Comment:
I think the `-settings` bit is redundant, given that all entries in a configuration file are settings. (Also, without the `-settings` bit, the `[bunch]` section will be automatically sorted to live before `[bunch-args]`.)